src/stac_utils/reach.py

```diff
@@ -0,0 +1,95 @@
+import logging
+import os, sys
+import json
+import pandas as pd
+import requests
+
+from datetime import date, timedelta
+from io import StringIO
+
+from stac_utils.google import auth_bq, run_query
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level = logging.INFO)
+
+if 'REACH_API_USER' not in os.environ:
+    # Try to load from .env
+    from dotenv import load_dotenv
+    state = 'local'
+    load_dotenv()
+
+try:
+    reach_keys = os.environ['REACH_API_USER']
+except KeyError:
+    sys.exit('Reach API keys environment variable not set, cannot run report')
+
+query_base = 'SELECT myv_van_id FROM '
+
+def get_api_headers():
+    auth_info = {
+        'username': os.environ['REACH_API_USER'],
+        'password': os.environ['REACH_API_PASSWORD']
+        }
+    r = requests.post('https://api.reach.vote/oauth/token', data = auth_info)
+    access_token = {
+        'Authorization': 'Bearer ' + json.loads(r.text)['access_token']
+        }
+    return access_token
+
+def update_people_to_tag(row, van_ids, tag_id, api_call_headers, remove=False):
```

Comment:
i think you can just delete most of what's below here (with the exception of the following comment), it was specific to the thing i was doing for KY